### ISSUE TITLE
Idea: Remove mango

### ISSUE THREAD

**Issue by jdonland** [HomebrewHomunculus/BrogueCE#23]
Replacing the mango with a slightly higher drop rate and slightly lower nutrition value for food rations seems like a simple minimalism win.

**Comment by aeosynth** [comment]
or you could keep the mango and tweak its stats, or just remove nutrition entirely: #25

**Comment by HomebrewHomunculus** [comment]
Interesting. I haven't played in a while so I didn't even remember what the mango does. (It gives only 1550 nutrition, which is 86% of the ration's 1800).

However, changing the nutrition values and drop rates of food might mess up a delicate balance. What do you find inelegant about the mango, is it just having to remember an extra item and how much nutrition it gives?

**Comment by jdonland** [comment]
Yeah, it's just odd that the food items are "ration" and "worse ration".
